srcpkgs/i3status-rust/template

```diff
@@ -1,18 +1,23 @@
 # Template file for 'i3status-rust'
 pkgname=i3status-rust
-version=0.22.0
-revision=2
+version=0.31.5
+revision=1
 build_style=cargo
 make_check_args="--bins"
 hostmakedepends="pkg-config"
-makedepends="dbus-devel pulseaudio-devel openssl-devel libsensors-devel"
+makedepends="dbus-devel pulseaudio-devel openssl-devel libsensors-devel pandoc"
```

Comment:
Why did you add `pandoc` to `makedepends` and not `hostmakedepends`? I'm surprised that this works at all to be honest.

Reply:
it didn't work. CI "passed" because it ended with pandoc's nocross being triggered and no packages were built

Reply:
Oh, haha, weird stuff.

@inalone you should put pandoc in hostmakedepends instead of makedepends then.